jbmc/README.md

```diff
@@ -27,10 +27,15 @@ JBMC compiles CBMC as part of its build process and as such has all the pre-requ
 Compilation
 ===========
 
-To compile you need to run the command:
+Before compilation, update git submodules using the command:
+
+```bash
+make -C jbmc/src setup-submodules
+```
```

Comment:
You'll also have to do a `make -C src minisat2-download` here

Reply:
I have added another commit to add this.